jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -445,6 +454,9 @@ public void testExecute_complex()
         buildAndRunComplex(
             targetImage, "testuser2", "testpassword2", localRegistry2, "pom-complex.xml"));
     assertWorkingDirectory("", targetImage);
+    assertEntrypoint(
+        "[java -Xms512m -Xdebug -cp /app/resources:/app/classes:/app/libs/* com.test.HelloWorld]",
```

Comment:
Can we have a test where `extraClasspath` is in action?

Reply:
🤦‍♂️ I clearly did something wrong here as this is clearly missing the extraClasspath.